guides/micronaut-rabbitmq-rpc/bookinventory/kotlin/src/main/kotlin/example/micronaut/BookInventory.kt

```diff
@@ -0,0 +1,7 @@
+package example.micronaut
+
+import io.micronaut.core.annotation.Creator
+import io.micronaut.core.annotation.Introspected
+
+@Introspected
+data class BookInventory @Creator constructor(val isbn: String, val stock: Int)
```

Comment:
Is `@Creator` necessary? If you want to include it, we should educate the user about it with a callout.